examples/todo/js/subscriptions/AddTodoSubscription.js

```diff
@@ -12,11 +12,8 @@ export default class AddTodoSubscription extends RelaySubscriptions.Subscription
   getSubscription() {
     return Relay.QL`subscription {
       addTodoSubscription {
-        clientMutationId
         todoEdge {
-          __typename
           node {
-            __typename
```

Comment:
It seems to work. There's a flash of the same edge being present twice, but that's there against master as well.